.phpcs.xml.dist

```diff
@@ -7,19 +7,19 @@
 
 	<arg value="sp"/>
 	<arg name="extensions" value="php"/>
+	<arg name="basepath" value="."/>
+	<arg name="parallel" value="8"/>
```

Comment:
Noting that PHPCS appears to have an [issue](https://github.com/squizlabs/PHP_CodeSniffer/issues/2304) with this arg being used on some installs of PHP 7.3.

It might be worth leaving this out for now, though it would only potentially affect contributors to WPCS.

Reply:
As this is the ruleset for WPCS itself, I'm not too concerned. If this (upstream) issue would affect a contributor, they can easily get round it by passing `--parallel=1` on the command line.

Correct me if I'm wrong, but I don't think changing this is a reason to delay the release.

Reply:
> I don't think changing this is a reason to delay the release.

Completely agree, which is why I approved it - just mentioning it for future reference for future readers.